gis/gdal-grass/Portfile

```diff
@@ -10,7 +10,7 @@ categories          gis
 # do not (re-)publish 1.4.3_0, 2.1.0_0, 2.1.0_1, 2.2.0_0, 2.2.0_1, or 2.2.0_2
 github.setup        OSGeo gdal-grass 1.0.2
 github.tarball_from archive
-revision            1
+revision            2
```

Comment:
Is this revision increase really needed?

Reply:
After fixing grass for 10.15 and earlier, we do want to reattempt the build of gdal-grass on the buildbot, but we can either schedule that build manually, or you can make *any* change to the gdal-grass Portfile to cause it to happen. For example, consider removing the `platforms darwin` line since that is now the default.

Reply:
Thanks for the explanation, I'm slowly starting to understand the MP infrastructure.
I removed the rev bump and instead removed `platforms darwin` for gdal-grass.